package/src/components/Message/MessageSimple/utils/renderText.tsx

```diff
@@ -133,7 +133,8 @@ export const renderText = <
   };
 
   const onLink = (url: string) => {
-    const pattern = new RegExp(/^\S+:\/\//);
+    // Matching these: https://reactnative.dev/docs/0.73/linking?syntax=ios#built-in-url-schemes
+    const pattern = new RegExp(/^(mailto:|tel:|sms:|\S+:\/\/)/);
```

Comment:
If you think it can work that way please feel free to do it. It would be great if we have all the logic confined to one place. 😄

Reply:
Actually looking a bit deeper into this, it seems that `linkifyjs` already takes care of all these things (i.e knows how to parse everything properly and prepends a protocol if one doesn't exist).

Same goes for e-mails. I actually now think that this regex should entirely be removed.

Reply:
[Here is the commit](https://github.com/GetStream/stream-chat-react-native/commit/9c67d44e) that introduced the change, since I don't have the context to figure out why it's been added I added the safeguard in the same spot. The other change in `Card.tsx` is long gone so we should be good.

Reply:
Also the tests are sufficient, just a small case I noticed that I added.